lib/application.js

```diff
@@ -168,11 +170,11 @@ Application.prototype.createClient = function () {
       connectionRetryTimeout: self.connectionRetryTimeout,
       desiredCapabilities: {
         browserName: 'electron',
-        chromeOptions: {
+        chromeOptions: Object.assign({
```

Comment:
To avoid overriding the values we provide this should probably be

```js
Object.assign({}, self.additionalChromeOptions, {
  binary: launcherPath,
  // Other stuff
})
```

Reply:
Not sure. It's a bit messy no matter which way around we do it: If a user sets "args" or "debuggerAddress" both ways, one is going to win. In the case the user sets "binary", the overwriting may, in fact, be intentional.